addons/address_autocomplete/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': "Address Autocomplete",
+    'summary': 'Assist with automatic completion & suggestions when filling address',
+    'version': '1.0',
+    'description': """
+This module Auto complete the address data.
+    """,
+    'category': 'Hidden/Tools',
+    'depends': ['base_setup'],
+    'data': [
+        'views/res_config_settings_views.xml',
+    ],
+    'auto_install': True,
```

Comment:
> have res.config setting to do so.

Now, since both the website_sale_autocomplete and the address_autocomplete module use the same api key google_places_api_key and res.config.settings, so which module should I install from res.config.settings? Should I install the website_sale_autocomplete, or the address_autocomplete.